paasta_tools/cli/cmds/exec.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+# Copyright 2015-2022 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""PaaSTA exec for humans"""
+import argparse
+import random
+import shlex
+import subprocess
+import sys
+
+from paasta_tools.cli.utils import figure_out_service_name
+from paasta_tools.cli.utils import guess_service_name
+from paasta_tools.cli.utils import lazy_choices_completer
+from paasta_tools.cli.utils import verify_instances
+from paasta_tools.utils import DEFAULT_SOA_DIR
+from paasta_tools.utils import list_clusters
+from paasta_tools.utils import list_services
+from paasta_tools.utils import PaastaColors
+
+
+def add_subparser(subparsers) -> None:
+    exec_parser = subparsers.add_parser(
+        "exec",
+        help="Execs into a running PaaSTA service",
+        description=(
+            "'paasta exec' works by running some k8s commands for you "
+            "in a human-friendly way."
```

Comment:
suggestion: it'd be nice to frame this in a way that would still be understandable to a new grad (e.g., something about letting you connect to a running container for x, y, z)